apps/webapp/src/components/shared/input-block.tsx

```diff
@@ -1,25 +1,17 @@
-import { Input, InputProps } from '@penumbra-zone/ui';
 import { cn } from '@penumbra-zone/ui/lib/utils';
 import { Validation, validationResult } from './validation-result.ts';
+import { ReactNode } from 'react';
 
-interface InputBlockProps extends InputProps {
+interface InputBlockProps {
   label: string;
-  className?: string;
-  inputClassName?: string;
-  validations?: Validation[];
-  value: string;
+  className?: string | undefined;
+  validations?: Validation[] | undefined;
```

Comment:
FWIW, needing to add `| undefined` to a prop that's already optional is another example of the downside of enforcing `exactOptionalPropertyTypes` in our tsconfigs. Obviously not a big problem, but just mentioning it since it's been a topic.

Reply:
It sounded like the majority of folks are interested in disabling `exactOptionalPropertyTypes`. Think we should feel free to and revisit it if it ends up not catching a bug in the future.